frontend/src/app/organization/organization-editor/organization-editor.component.css

```diff
@@ -1,17 +1,48 @@
-.mat-mdc-card {
-  margin: 1em;
-  max-width: 640px;
+mat-card {
+  margin: 2em;
 }
 
-.mat-mdc-form-field {
-  width: stretch;
+mat-card-header {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 20px;
 }
 
-.mat-mdc-card-actions .mdc-button {
-  margin-left: 8px;
-  margin-bottom: 8px;
+mat-card-title {
+  flex: 1;
+  text-align: center;
 }
 
 mat-card-content {
-  margin-top: 8px;
-}
\ No newline at end of file
+  display: flex;
+  flex-wrap: wrap;
+  gap: 16px;
+  justify-content: space-between;
+}
+
+mat-form-field {
+  flex: 0 0 calc(50% - 16px);

```

Comment:
Might be a good idea to find a workaround to using `calc`! This could work w/ flex boxes, where each column gets `width: 50%`.